.github/workflows/verification.yml

```diff
@@ -7,26 +7,18 @@ on:
       - master
 
 jobs:
-  tests-lint:
-    name: Run tests and lint
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-        with:
-          fetch-depth: "0"
-      - name: Unit Tests
-        uses: cedrickring/golang-action@1.7.0
-        env:
-          GO111MODULE: "on"
-        with:
-          args: make install && make test_unit
-      - name: Lint
-        uses: cedrickring/golang-action@1.7.0
-        env:
-          GO111MODULE: "on"
-        with:
-          args: make install && make lint
+  lint:
+    name: Lint
+    uses: flyteorg/flytetools/.github/workflows/lint.yml@master
+    with:
+      go-version: 1.16
```

Comment:
We are using go version 1.17. Also should we have the default coming from lint.yml and only override if the repo doesn't support the default version

Reply:
cc : @evalsocket

Reply:
I will create a follow up PR for go 1.17 upgrade